.pre-commit-config.yaml

```diff
@@ -1,11 +1,11 @@
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v1.4.0
-  hooks:
-  - id: trailing-whitespace
-    language_version: python3.6
-- repo: https://github.com/ambv/black
-  rev: 18.6b4
-  hooks:
-  - id: black
-    language_version: python3.6
\ No newline at end of file
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Looks like the root element shifted two spaces, is this yaml still valid?

Reply:
Weird. Checking my linter.

Reply:
It's valid @dragonmantank